src/api/handlers/charts.go

```diff
@@ -20,6 +20,7 @@ func GetChart(params operations.GetChartParams, c data.Charts) middleware.Respon
 		return notFound(chartResourceName)
 	}
 	chartResource := helpers.MakeChartResource(chart, params.Repo)
+	helpers.AddLatestLinks(chartResource, *chart.Version)
```

Comment:
The API doesn't currently return non-versioned chart resources. We should consider doing that separately:

https://github.com/helm/monocular/issues/41